_posts/hannahlwang/2016-01-14-hannahlwang-turtle-program.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: post
+author: hannahlwang
+title: Hannah's Turtle Exercise
+---
+Here's my custom input turtle:
+<iframe src="https://trinket.io/embed/python/2abfbccc1e" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+Here's my turtle hack:
+<iframe src="https://trinket.io/embed/python/27186f1f20" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+I decided to write a program in Turtle to make a cake, with custom input. At the end, Tina the turtle eats a slice. Is there some way to make Tina bigger at the end?? I tried (and failed) to figure that out.
```

Comment:
Good question.  Trinket actually doesn't have good support for it but there is a way in real Python: https://docs.python.org/2/library/turtle.html#turtle.turtlesize  More on reading docs later in the semester